rotkehlchen/db/dbhandler.py

```diff
@@ -732,6 +734,18 @@ def delete_aave_data(self) -> None:
         self.conn.commit()
         self.update_last_write()
 
+    def delete_uniswap_data(self) -> None:
+        """Delete all historical Uniswap data"""
+        cursor = self.conn.cursor()
+        cursor.execute(
+            f'DELETE FROM trades WHERE location="{Location.UNISWAP.serialize_for_db()}";',
+        )
+        cursor.execute(
+            f'DELETE FROM used_query_ranges WHERE name LIKE "{UNISWAP_TRADES_PREFIX}%";',
```

Comment:
you would need to do something similar for the events here with a different prefix. Just a reminder.